src/utilities/PostGresReminder.ts

```diff
@@ -0,0 +1,139 @@
+import { Pool } from "pg";
+import {
+    EmbedBuilder,
+    TextChannel,
+} from "discord.js";
+import { Bot } from "../Bot";
+
+const pool = new Pool({
+    user: "postgres",
+    password: "a",
+});
+ 
+type SearchUserType = { id: number, message: string, alert_time: Date; }[] 
+type SearchTimeType = { id: number, message: string, user_id: string, channel_id: string; }[]
+
+export namespace PostGresReminder {
+    // creates table for storing alerts 
+    export async function createAlertTable () {
```

Comment:
Explicitly specify the return type for functions.

```suggestion
    export async function createAlertTable(): Promise<void> {
```